xml/public-cloud-setup.xml

```diff
@@ -114,18 +123,42 @@
       <itemizedlist mark="bullet" spacing="normal">
        <listitem>
         <para> Obtain external IP-address from within Amazon EC2 instance </para>
-        <screen>ec2metadata --public-ipv4</screen>
+        <screen>$ ec2metadata --public-ipv4</screen>
        </listitem>
        <listitem>
         <para> Obtain external IP-address from within Google Compute Engine
          instance </para>
-        <screen>gcemetadata --query instance --external-ip</screen>
+        <screen>$ gcemetadata --query instance --external-ip</screen>
        </listitem>
        <listitem>
         <para> Obtain external IP-address from within Microsoft Azure instance </para>
-        <screen>azuremetadata --external-ip</screen>
+        <screen>$ azuremetadata --external-ip</screen>
+       </listitem>
+      </itemizedlist>
+     </listitem>
+
+     <listitem>
+      <para>Update the DNS entry for this instance within the DNS service of your
+       VPC (Virtual Private Cloud) environment, refer to the cloud service provider
+       documentation for detailed instructions:
+       <itemizedlist mark="bullet" spacing="normal">
+       <listitem>
+        <para><link
+         xlink:href="http://docs.aws.amazon.com/AmazonVPC/latest/UserGuide/vpc-dns.html">
+         DNS setup on Amazon EC2</link></para>
+       </listitem>
+       <listitem>
+        <para><link
+         xlink:href="https://cloud.google.com/compute/docs/storing-retrieving-metadata">
```

Comment:
Is this the correct link? I don't see anything DNS related on that page. Maybe https://cloud.google.com/dns/overview instead ?